src/main/python/ayab/engine/control.py

```diff
@@ -191,13 +227,13 @@ def cnf_line_API6(self, line_number):
         else:
             return True  # pattern finished
 
-    def __update_status(self, line_number, color, bits):
+    def __update_status(self, line_number:int, color:int, bits:bitarray)->None:
         self.status.current_row = self.pat_row + 1
         self.status.line_number = line_number
         if self.inf_repeat:
             self.status.repeats = self.pattern_repeats
         if self.mode != Mode.SINGLEBED:
-            self.status.color_symbol = self.COLOR_SYMBOLS[color]
+            self.status.color_symbol = self.COLOR_SYMBOLS[color] #type: ignore
```

Comment:
there's prolly a way to get this to work but I think it'd require a weird typing for color in the argument definition above.